config.json

```diff
@@ -331,7 +331,7 @@
           "arrays"
         ],
         "prerequisites": [
-          "arrays"  
```

Comment:
I didn't notice this one before committing. Should I remove it?

Reply:
it's fine.